writer.go

```diff
@@ -1013,13 +1013,13 @@ func (c *writerColumn) WritePage(page Page) (numValues int64, err error) {
 				})
 				return numValues, err
 
-			case CompressedPage:
+				//case CompressedPage:
```

Comment:
?

Reply:
See the comment below explaining why this code is disabled for now.